jenkins-jobs/pipelines/release-pipeline.groovy

```diff
@@ -427,13 +427,13 @@ node('Slave') {
         stage('Prepare release') {
             dir(DEBEZIUM_DIR) {
                 sh "git checkout -b $CANDIDATE_BRANCH"
-                sh "mvn clean install -DskipTests -DskipITs -Poracle"
+                sh "mvn clean install -DskipTests -DskipITs -Poracle-xstream"
```

Comment:
@Naros Are here missing commas?

Reply:
Nope, the new `oracle-stream` profile is required to build and include the Xstream bits as they're excluded by default.

Reply:
Sorry, I misread dash as minus to disable `xstream` profile